code/go/internal/validator/semantic/validate_minimum_kibana_version.go

```diff
@@ -0,0 +1,153 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package semantic
+
+import (
+	"regexp"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/pkg/errors"
+
+	ve "github.com/elastic/package-spec/v2/code/go/internal/errors"
+	"github.com/elastic/package-spec/v2/code/go/internal/fspath"
+	"github.com/elastic/package-spec/v2/code/go/internal/pkgpath"
+)
+
+// ValidateMinimumKibanaVersion if the package is an input package, and the package version is >= 1.0.0,
+// then the kibana version condition must be >= 8.8.0
+func ValidateMinimumKibanaVersion(fsys fspath.FS) ve.ValidationErrors {
+	manifest, err := readManifest(fsys)
+	if err != nil {
+		return ve.ValidationErrors{err}
+	}
+
+	packageType, err := getPackageType(*manifest)
+	if err != nil {
+		return ve.ValidationErrors{err}
+	}
+
+	packageVersion, err := getPackageVersion(*manifest)
+	if err != nil {
+		return ve.ValidationErrors{err}
+	}
```

Comment:
Ah I missed this util, thanks!

Reply:
Cyclic dependency prevents me from doing this unfortunately

Reply:
An option to break this cyclic dependency could be move all the `code/go/internal/validator/package*.go` files to its own package. For instance it could be created a new package like `code/go/internal/packages/` and update all the references.